apps/testing-javascript/src/data/sanity-import.ts

```diff
@@ -71,6 +71,25 @@ const CastingWordsSchema = z
     srt: z.string(),
   })
   .nullish()
+  .transform((data) => {
+    if (data === null || data === undefined) {
+      return data
+    }
+
+    const {srt, ...rest} = data
+
+    if (srt.startsWith('WEBVTT\n\n')) {
+      const updatedSrt = srt.replace(/^WEBVTT\n\n/, '')
```

Comment:
I originally wrote a separate script back when we realized that the `WEBVTT` needed to be scrubbed from the SRT text. It seemed worthwhile to pull those changes back into this script to make it a one-step process if this ever needs to get run again.